### PR TITLE
ui: add missing Arabic translations & show in UI

### PR DESCRIPTION
**Pull request by royjr**
- [X] Removed vanished
- [X] Added unfinished translations
- [X] Cycled offroad events
- [X] test_translations.py
- [X] added arabic next to turkish assuming sorting by language regions
- [X] C3 visual check
```
Updating 'selfdrive/ui/translations/tmp/main_ar.ts'...
Found 272 source text(s) (0 new and 272 already existing)
```

Screenshot 2023-09-11 at 8 41 07 PM

### CODE REVIEW

**Comment by sshane** [comment]
If you can, can you test on a C3/3X to make sure that the font looks good? It previously didn't for Thai so we added a few at the same time.

**Comment by royjr** [comment]
Looks just as good 👍

File: selfdrive/ui/translations/main_ar.ts [hunk 1]
```diff
@@ -1047,36 +1071,84 @@ location set</source>
...
+    </message>
+    <message>
+        <source>Navigate on openpilot</source>
+        <translation>التنقل على openpilot</translation>
+    </message>
+    <message>
+        <source>When navigation has a destination, openpilot will input the map information into the model. This provides useful context for the model and allows openpilot to keep left or right appropriately at forks/exits. Lane change behavior is unchanged and still activated by the driver. This is an alpha quality feature; mistakes should be expected, particularly around exits and forks. These mistakes can include unintended laneline crossings, late exit taking, driving towards dividing barriers in the gore areas, etc.</source>
+        <translation>عندما يكون للملاحة وجهة، سيقوم openpilot بإدخال معلومات الخريطة في النموذج. يوفر هذا سياقًا مفيدًا للنموذج ويسمح لـ openpilot بالبقاء يسارًا أو يمينًا بشكل مناسب عند المخارج/المخارج. لا يتغير سلوك تغيير المسار ولا يزال السائق ينشطه. هذه ميزة جودة ألفا؛ وينبغي توقع الأخطاء، وخاصة حول المخارج والشوك. يمكن أن تشمل هذه الأخطاء عبور الخطوط غير المقصودة، والخروج المتأخر، والقيادة نحو الحواجز الفاصلة في المناطق الممتلئة بالدماء، وما إلى ذلك.</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_ar.ts:1127]: Apparently this literally translates as "blood-soaked areas" :p

**Comment by sshane** [comment]
`المناطق الممتلئة بالدماء`
is still there, are you sure that's fine?

**Review by sshane — APPROVED** [review]

**Comment by sshane** [comment]
thanks!

**Pull request merged**